class_generator/tests/manifests/ServiceMeshMember/service_mesh_member.py

```diff
@@ -0,0 +1,40 @@
+# Generated using https://github.com/RedHatQE/openshift-python-wrapper/blob/main/scripts/resource/README.md
+
+from typing import Any, Dict, Optional
+from ocp_resources.resource import NamespacedResource, MissingRequiredArgumentError
+
+
+class ServiceMeshMember(NamespacedResource):
+    """
+    No field description from API; please add description
```

Comment:
Please add description :)

Reply:
but then it will fail - this is the tests part so we should not modify it

Reply:
haaa, it's under tests ......